package.json

```diff
@@ -26,7 +26,6 @@
     "@octokit/core": "^4.0.0",
     "@octokit/oauth-authorization-url": "^5.0.0",
     "@octokit/oauth-methods": "^2.0.0",
-    "@types/aws-lambda": "^8.10.83",
```

Comment:
Should we updatr `package-lock.json` accordingly?

Reply:
I will fix this.